common/dynamicconfig/constants.go

```diff
@@ -2852,6 +2852,13 @@ const (
 	// Default value: 30 seconds
 	HistoryGlobalRatelimiterGCAfter
 
+	// LocalPollWaitTime is the wait time for a poller to wait before considering request forwarding
+	// KeyName: matching.localPollWaitTime
+	// Value type: Duration
+	// Default value: 10ms
+	// Allowed filters: domainName, taskListName, taskListType
+	LocalPollWaitTime
```

Comment:
Is there a strategy for this key?
Will it be the default set for all clusters? Should it be set for other customers in OS?

Reply:
Yeah, use this default value for all clusters, as this is better than 0ms.